### ISSUE TITLE
The "submit an issue" button on page 4 on the iBooks preview is too difficult to click.

### ISSUE THREAD

**Issue by mrpitchfork** [scottgarner/makingthingsbeautiful#1]
The clickable area is just about the width of the "i" rather than the width of the whole circle.  I was clearly able to get here, but it took me a while to realize there was a functioning link.

The approximate clickable area is highlighted in cyan, where I think the clickable area should be in magenta.

![screenshot at dec 06 12-26-30](https://cloud.githubusercontent.com/assets/1261523/20942227/51650082-bbaf-11e6-9fe6-f3a42683dbd8.png)

**Comment by scottgarner** [comment]
You're absolutely right. The version of iBooks Author I used when building the book several years ago had some very strange limitations on what could and could not be a link. In this case, the letter "i" could be an external link, but not the circle around it. 

I think those issues have been fixed in the software. I'd like to revisit the book at some point and overhaul a lot of similar problems, but don't have a definite timeline for doing so.
